src/test/SystemTest/docker/kafka-connect/Dockerfile

```diff
@@ -2,4 +2,4 @@
 #FROM confluentinc/cp-kafka-connect-base:latest # latest build doesn't even start correctly without our connector
 FROM confluentinc/cp-kafka-connect:5.5.1-1-ubi8
 RUN mkdir -p /etc/kafka-connect/jars/kafka-connect-scalyr-sink
-COPY target/kafka-connect-scalyr-sink-0.1-package/share/java/kafka-connect-scalyr-sink/ /etc/kafka-connect/jars/kafka-connect-scalyr-sink
+COPY target/kafka-connect-scalyr-sink-*-package/share/java/kafka-connect-scalyr-sink /etc/kafka-connect/jars/kafka-connect-scalyr-sink
```

Comment:
This seems like it could be brittle.  If you haven't done a `mvn clean` to remove old builds from `target`, you could match multiple packages here.

Is there any chance you can change `mvn` so that it also writes the target to a stable name, like `kafka-connect-scalyr-sink-latest-package`?

Reply:
The package name will only change when we change versions, which is not very often.  It is possible to add a link to from `version` to `latest` much like how the ScalyrSite InternalTools does it.